runtime/src/main/java/com/dylibso/chicory/runtime/Machine.java

```diff
@@ -16,10 +16,6 @@ public class Machine {
 
     private static final System.Logger LOGGER = System.getLogger(Machine.class.getName());
 
-    public static final double TWO_POW_63_D = 0x1.0p63; /* 2^63 */
-
-    public static final float TWO_POW_64_PLUS_1_F = 1.8446743E19F; /* 2^64 + 1*/
```

Comment:
These were unused right? I think @thomasdarimont added them. Maybe we refactored out and forgot to remove? Some linting on unused code might be helpful here.

Reply:
`TWO_POW_63_D` was used in the old code. `TWO_POW_64_PLUS_1_F` was unused.